app/controllers/avo/actions_controller.rb

```diff
@@ -7,6 +7,8 @@ class ActionsController < ApplicationController
     before_action :set_action, only: [:show, :handle]
 
     def show
+      model = @resource.class.find_scope.find params[:id]
```

Comment:
I think `params[:id]` here is always nil, am I wrong?

Reply:
I mean, in show view it's ok but in other use cases this will break, maybe we should protect it

Reply:
Yes quite possibly! This is just some code i threw together quickly and I don’t have many examples of actions to test it on, it may well have bugs

Reply:
I think there's value in this approach. There's [this](https://github.com/avo-hq/avo/pull/1206) PR that tackles the same issue in a different way. 
Yes, it will work only for one record (from show view), but that's ok. We'll soon introduce row-only actions so this will come in handy. 

I'd like to have a test here to protect this behavior in the future.